src/assets/stylesheets/_variables.scss

```diff
@@ -0,0 +1,12 @@
+$brand-black: (
+  10: #000,
```

Comment:
rather than naming with number can we use `primary`, `light`, `lighter`, `dark`, `darker` type of naming convention?

Reply:
Updated in bddc3f6bbc0d01e9139ed35d9e27960af759be5b. Not sure if this is ok for you.